### PR TITLE
Feature/ Badges

### PR DESCRIPTION
**Pull request by SalonSikdar**
Build the badge on avatars and badge on icons features. added docs for the same

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Code is clean & readable.
It's good to merge !

File: pages/components/Badge/badge.css [hunk 1]
```diff
@@ -36,3 +36,98 @@
   --bs-bg-opacity: 1;
   background-color: var(--warning-color) !important;
 }
+
+.badge-round {
+  color: var(--white-global);
+  border-radius: 10%;
+  width: 70px;
+  position: inherit;
+  top: 20px;
```
Comment (govindappaarun) [inline pages/components/Badge/badge.css:45]: any reasons why badge round has to be at this position?

File: pages/components/Badge/badge.css [hunk 2]
```diff
@@ -36,3 +36,98 @@
   --bs-bg-opacity: 1;
   background-color: var(--warning-color) !important;
 }
+
+.badge-round {
+  color: var(--white-global);
+  border-radius: 10%;
+  width: 70px;
```
Comment (harshkotkar5557) [inline pages/components/Badge/badge.css:43]: Try to use rem instead of pixels

**Review by harshkotkar5557 — APPROVED** [review]
Looks good but try to used rem instead of pixels

File: pages/components/Badge/badge.css (hunk 2 above)
Comment (SalonSikdar) [reply pages/components/Badge/badge.css:43]: Ack, done

File: pages/components/Badge/badge.css (hunk 1 above)
Comment (SalonSikdar) [reply pages/components/Badge/badge.css:45]: made square badges first. 
then round. seperation of concern, no other reason

**Pull request merged**